UserService/src/models/user.ts

```diff
@@ -0,0 +1,25 @@
+import mongoose from "mongoose";
+
+const userSchema = new mongoose.Schema<IUser>({
+  username: {
+    type: String,
+    required: true
+  },
+  password: {
+    type: String,
+    required: true
+  },
+  imgUrl: {
+    type: String,
+    required: false
+  }
+})
+
+export default mongoose.model("User",userSchema)
+
+export interface IUser {
+  username: string,
+  password: string,
+  imgUrl?: string
+  id: mongoose.Schema.Types.ObjectId
+}
```

Comment:
Does demo need to provide the field of our two actors (customer and owner)?
if not, do it later.
ที่เหลือก็เหมือนพี